programs/applications/calculator/CalculatorWidget.cpp

```diff
@@ -150,6 +152,18 @@ void CalculatorWidget::do_op(const std::string& op) {
 	}
 }
 
+void CalculatorWidget::disp_error() {
+	display->set_label("Can't divide by zero.");
```

Comment:
Looks like this error message is a little long to fit in the default size of the calculator window. Could we change this to something like `NaN`?

Reply:
@byteduck updated to NaN.